src/Directory.Packages.props

```diff
@@ -0,0 +1,106 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
+  </PropertyGroup>
+  <ItemGroup>
+    <GlobalPackageReference Include="Nerdbank.GitVersioning" Version="3.7.115" PrivateAssets="All" />
+    <GlobalPackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.3.4"  PrivateAssets="All" />
+    <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All"/>
+    <GlobalPackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.13.2" PrivateAssets="All"/>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageVersion Include="Azure.Bicep.Internal.RoslynAnalyzers" Version="0.1.45" />
+    <PackageVersion Include="Azure.Bicep.Types" Version="0.5.110" />
+    <PackageVersion Include="Azure.Bicep.Types.Az" Version="0.2.789" />
+    <PackageVersion Include="Azure.Bicep.Types.K8s" Version="0.1.644" />
+    <PackageVersion Include="Azure.Containers.ContainerRegistry" Version="1.1.1" />
+    <PackageVersion Include="Azure.Deployments.ClientTools" Version="1.384.0" />
+    <PackageVersion Include="Azure.Deployments.Engine" Version="1.424.0" />
+    <PackageVersion Include="Azure.Deployments.Extensibility.Core" Version="0.1.67" />
+    <PackageVersion Include="Azure.Deployments.Internal.GenerateNotice" Version="0.1.45" />
+    <PackageVersion Include="Azure.Deployments.Templates" Version="1.424.0" />
+    <PackageVersion Include="Azure.Identity" Version="1.13.2" />
+    <PackageVersion Include="Azure.ResourceManager.ResourceGraph" Version="1.0.1" />
+    <PackageVersion Include="Azure.ResourceManager.Resources" Version="1.9.0" />
+    <PackageVersion Include="BenchmarkDotNet" Version="0.14.0" />
+    <PackageVersion Include="CommandLineParser" Version="2.9.1" />
+    <PackageVersion Include="DiffPlex" Version="1.7.2" />
+    <!-- Version 8 or above are no longer fully open-source -->
+    <PackageVersion Include="FluentAssertions" Version="7.2.0" />
```

Comment:
>7.2.0 [](http://example.com/codeflow?start=56&length=5)

Do we need to configure DependaBot to stop upgrading this one?

Reply:
It's already there - https://github.com/Azure/bicep/blob/27cc8db2ed40340963bd15733e8e425b865fdc1d/.github/dependabot.yml#L31